packages/select-rich/src/LionSelectRich.js

```diff
@@ -90,6 +89,13 @@ export class LionSelectRich extends ScopedElementsMixin(
 
       name: {
         type: String,
+        reflect: true,
+      },
+
+      // Copied from LionField
+      submitted: {
```

Comment:
That's why it should be put in InteractionStsteMixin :)

Reply:
True, I guess I'll just fix that in the other PR? :P

Reply:
Actually I'll fix it here